src/test/scripts/kind/deploy_app.sh

```diff
@@ -43,10 +43,7 @@ deploy_app() {
   fi
   sed -i "s/DC_APP_REPLACEME/${DC_APP}/g" ../../../test/config/kind/common-values.yaml
   sed -i "s/DB_TYPE_REPLACEME/${DB_TYPE}/g" ../../../test/config/kind/common-values.yaml
-  if [ ${DC_APP} == "bitbucket" ]; then
-    # this is to test mesh
-    IMAGE_OVERRIDE="--set image.tag=8.6.1"
```

Comment:
Was this due to the mesh?

Reply:
Yes, leftovers. We test mesh in a separate Bamboo plan now.